lua/telescope/builtin/internal.lua

```diff
@@ -993,11 +993,46 @@ internal.colorscheme = function(opts)
 end
 
 internal.marks = function(opts)
-  local marks = vim.api.nvim_exec("marks", true)
-  local marks_table = vim.fn.split(marks, "\n")
-
-  -- Pop off the header.
-  table.remove(marks_table, 1)
+  local bufnr = opts.bufnr or vim.api.nvim_get_current_buf()
```

Comment:
this can be omitted: https://github.com/nvim-telescope/telescope.nvim/blob/39b12d84e86f5054e2ed98829b367598ae53ab41/lua/telescope/builtin/init.lua#L458

Reply:
Thanks, I fixed it.